data/channels.csv

```diff
@@ -18152,259 +18152,259 @@ LiyuanTV.cn,Liyuan TV,,,,CN,,Huzhou,c/CN,zho,general,FALSE,,,,,https://i.imgur.c
+LocoTV.nl,Loco TV,,,Loco Media Group,NL,,Wezep,c/NL,nld,general,FALSE,,,,https://www.locomediagroep.nl/contact,https://i.imgur.com/QqRAmg0.png
+LOETV.nl,LOE TV,,,,NL,,Elburg,c/NL,nld,general,FALSE,,,,https://loemedia.nl/contact,https://i.imgur.com/8i6OwaS.png
 LofTelevision.cl,Lof Television,,Lof,,CL,,,c/CL,spa,,FALSE,,,,https://lofchile.cl/,https://i2.paste.pics/f481e57821baed274d68c5b0c954cd96.png
-LogoEast.us,Logo East,,,,US,,,c/US,eng,,FALSE,,,,http://www.logotv.com/,https://upload.wikimedia.org/wikipedia/commons/thumb/2/2f/Logo_TV_2015.svg/512px-Logo_TV_2015.svg.png
+LogoEast.us,Logo East,,,Paramount Media Networks,US,US-NY,New York City,c/US,eng,entertainment,FALSE,2005-01-01,,,http://www.logotv.com/,https://upload.wikimedia.org/wikipedia/commons/thumb/2/2f/Logo_TV_2015.svg/512px-Logo_TV_2015.svg.png
 LogoPlutoTV.us,Logo Pluto TV,,Pluto TV,,US,,,c/US,eng,entertainment,FALSE,,,,https://pluto.tv/en/live-tv/logo-pluto-tv,https://i.imgur.com/GtNgJ28.png
-LOGOSChannel.cm,LOGOS Channel,,,,CM,,,c/CM,fra,religious,FALSE,,,,,https://i.ibb.co/KW8pBSj/Screenshot-20230807-140218.jpg
-LogosTV.es,Logos TV,,,,ES,,,c/ES,spa,,FALSE,,,,,https://i.imgur.com/Kr3U3Ub.png
-LogosTV.us,Logos TV,,,,US,,,c/US,ara,religious,FALSE,,,,https://logoschannel.com/,https://i.imgur.com/h8DioXU.png
-LogosTVEnglish.us,Logos TV English,,,,US,,,c/US,eng,religious,FALSE,,,,,https://i.imgur.com/LDfEK81.png
-LogosTVEstudio.es,Logos TV Estudio,,,,ES,,,c/ES,spa,religious,FALSE,,,,https://logostv.es/,https://i.imgur.com/qkHrttj.png
-LogosTVKids.es,Logos TV Kids,,,,ES,,,c/ES,spa,kids,FALSE,,,,https://logostv.es/,https://i.imgur.com/Kr3U3Ub.png
-LogosTVSalud.es,Logos TV Salud,,,,ES,,,c/ES,spa,lifestyle,FALSE,,,,https://logostv.es/,https://i.imgur.com/qkHrttj.png
-LogoWest.us,Logo West,,,,US,,,c/US,eng,,FALSE,,,,http://www.logotv.com/,https://upload.wikimedia.org/wikipedia/commons/thumb/2/2f/Logo_TV_2015.svg/512px-Logo_TV_2015.svg.png
-LohasChannel.tw,Lohas Channel,樂活頻道,,Lexi International Co. Ltd.,TW,,,c/TW,jpn;eng,xxx,TRUE,,,,http://www.goldentv.com.tw/index.php?m=Product&a=index&pc_code=1080113&pop=false,https://i.imgur.com/mXxR9RN.png
-LoHermidaTelevision.cl,Lo Hermida Television,Lo Hermida Televisión,,,CL,CL-RM,,s/CL-RM,spa,,FALSE,,,,https://lhtv.cl/,https://i2.paste.pics/2cfcf32766eb75a8fec96086bc741ec2.png
-LokaleOmroepZeewolde.nl,Lokale Omroep Zeewolde,LOZ,,,NL,,,c/NL,nld,general,FALSE,,,,https://www.lokaleomroepzeewolde.nl/,https://i.imgur.com/Dw4u9wT.png
+LOGOSChannel.cm,LOGOS Channel,,,,CM,,Yaounde,c/CM,fra,religious,FALSE,,,,https://www.facebook.com/LaParoleEcrite,https://i.ibb.co/KW8pBSj/Screenshot-20230807-140218.jpg
+LogosTV.us,Logos TV,,,,US,US-MD,Columbia,c/US;c/ES,spa,religious,FALSE,,,,https://logostv.es/,https://i.imgur.com/Kr3U3Ub.png
+LogosNorthridgeTV.us,Logos Northridge TV,,,,US,US-CA,Northridge,c/US,ara,religious,FALSE,,,,https://logoschannel.com/,https://i.imgur.com/h8DioXU.png

```

Comment:
```suggestion
LogosTV.es,Logos TV,,,,ES,,,c/ES,spa,,FALSE,,,,,https://i.imgur.com/Kr3U3Ub.png
LogosTV.us,Logos TV,,,,US,US-CA,Northridge,c/US,ara,religious,FALSE,,,,https://logoschannel.com/,https://i.imgur.com/h8DioXU.png
```

Incorrect channel name. Be sure the channel's name matches on what is being broadcast.
Two instances of LogosTV exists. One is the Spain, another is from US.

Reply:
I did: theyre both called Logos. Edit them as you wish.

Reply:
Edited